src/adapters/catalonia.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * This code is responsible for implementing all methods related to fetching
+ * and returning data for the Catalonian data sources. credit to @magsyg
+ * based off of openaq-fetch PR #711 
+ */
+
+'use strict';
+
+import { REQUEST_TIMEOUT } from '../lib/constants.js';
+import { default as baseRequest } from 'request';
+import { default as moment } from 'moment-timezone';
```

Comment:
This still needs to be updated to the luxon

Reply:
This is one that I was not able to reproduce what magsyg did with moment in luxon

Reply:
figured it out, added 'no, nox, pm1' to enum, acceptableParameters